.gitattributes

```diff
@@ -1,9 +1,13 @@
-.editorconfig export-ignore
-.gitattributes export-ignore
-.gitignore export-ignore
-.php-cs-fixer.php export-ignore
-phpunit.xml export-ignore
-tests export-ignore
-git-hooks export-ignore
-bin/add-git-hooks export-ignore
-.github export-ignore
+/bin/ export-ignore
+/CHANGELOG.md export-ignore
+/CONTRIBUTING.md export-ignore
+/.editorconfig export-ignore
+/.gitattributes export-ignore
+/git-hooks/ export-ignore
+/.github/ export-ignore
+/.gitignore export-ignore
+/.php-cs-fixer.php export-ignore
+/phpstan.neon export-ignore
+/phpunit.xml export-ignore
+/tests/ export-ignore
+/UPGRADE.md export-ignore
```

Comment:
I'd also leave `/UPGRADE.md` in

Reply:
Both done 🍏

Reply:
I let you count 🙃 how many users open up the vendor directory to read documents.